prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Jorge Anguiano';
+    console.log(fullName);
+    var numberOfPets = '1';
```

Comment:
Hi Jorge, be sure to leave out the quotes on numbers so that the browser doesn't interpret them as strings.